geo/geo_computation.py

```diff
@@ -23,16 +23,16 @@
 def main(argv):
     del argv  # Unused.
 
-    # Compute graph over Manhattan
+    # Compute graph over Manhattan.
     graph = ox.graph_from_place('Manhattan, New York City, New York, USA')
 
     # Convert a graph to nodes and edge GeoDataFrames.
     nodes, _ = ox.graph_to_gdfs(graph)
 
     print(
         compute_route(
```

Comment:
Let's use the latitude then longitude convention:

https://en.wikipedia.org/wiki/ISO_6709

Reply:
Resolved in chat -- we'll use this for now because y is conventionally used for latitude and x for longitude. We may want to subclass Point as GeoPoint or something like that.